client/guidestours/style.scss

```diff
@@ -58,3 +84,47 @@ $zoom-scale: 5; // the multiplier determining the size of the animated rings
 .guidestours__pointer__ring:after {
 	animation-delay: #{ $animation-speed / 4 };
 }
+
+.guidestours__buttonrow__choice {
+	button {
+		width: 48%;
+	}
+	button:nth-child(1) {
+		margin-right: 4%;
+	}
+	button.guidestours__button__secondary {
+		background: $gray-light;
```

Comment:
This is the "No, thanks." button of the first tour step only to make it more visible that users don't *have* to take the tour. Maybe @shaunandrews has more to add?